docs/docs/top-k.md

```diff
@@ -26,7 +26,7 @@ Data flow is the incoming game scores. Flow id is the user id, and the value is
 Every time a user scores points, they're added to the Top K list. 
 
 - If the result is `nil`, the user is already in the Top K. Update the sorted set with the user's new score. 
-- if the result is an id of another player, the player you just added (`id1`) took over the player who got returned (`id2`). Remove player `id2` from the sorted set and add player `id1`. 
+- If the result is an id of another player, the player you just added (`id1`) took over the player who got returned (`id2`). Remove player `id2` from the sorted set and add player `id1`. 
```

Comment:
Why do you need to maintain a separate sorted set? Why not call `TOPK.LIST`?

Reply:
I didn't actually write this part and was just editing the if -> If, but can definitely reword this bit as well.

Reply:
Seems like the idea was to maintain both a top-k and a full on leaderbaord with a sorted set so that no matter how many players are added, you can still see their scores. If you're not in the top-k players, I don't think there's a call to retrieve your score. Well, there's `count` but it's deprecated so I definitely don't want to recommend that in the docs.

Reply:
Well, I guess it is not a good use case for top-k if you want to retrieve the score. As for the current example - it seems redundant to use top-k as well.